File: cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/setup/PrepareImageRequest.java
package com.sequenceiq.cloudbreak.cloud.event.setup;

import com.sequenceiq.cloudbreak.cloud.event.context.CloudContext;
import com.sequenceiq.cloudbreak.cloud.event.resource.CloudStackRequest;
import com.sequenceiq.cloudbreak.cloud.model.CloudCredential;
import com.sequenceiq.cloudbreak.cloud.model.CloudStack;

public class PrepareImageRequest<T> extends CloudStackRequest<PrepareImageResult> {
    public PrepareImageRequest(CloudContext cloudContext, CloudCredential cloudCredential, CloudStack cloudStack) {
        super(cloudContext, cloudCredential, cloudStack);
    }
}

File: cloud-reactor-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/setup/PrepareImageRequest.kt
package com.sequenceiq.cloudbreak.cloud.event.setup

import com.sequenceiq.cloudbreak.cloud.context.CloudContext
import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformRequest
import com.sequenceiq.cloudbreak.cloud.model.CloudCredential
import com.sequenceiq.cloudbreak.cloud.model.CloudStack
import com.sequenceiq.cloudbreak.cloud.model.Image

class PrepareImageRequest<T>(cloudContext: CloudContext, cloudCredential: CloudCredential, val stack: CloudStack, val image: Image) : CloudPlatformRequest<PrepareImageResult>(cloudContext, cloudCredential)
